Extension/package.json

```diff
@@ -43,6 +43,33 @@
   ],
   "main": "./dist/main",
   "contributes": {
+    "taskDefinitions": [
+      {
+        "type": "cppbuild",
+        "required": [
+          "command",
+          "label"
+        ],
+        "properties": {
+          "label": {
+            "type": "string",
+            "description": "Task's name."
+          },
+          "command": {
+            "type": "string",
+            "description": "The compiler path."
+          },
+          "args": {
+            "type": "array",
+            "description": "Additional compiler args."
+          },
+          "options": {
```

Comment:
What is `options`?

Reply:
some build options like "cwd" and "env" goes to this part.

Reply:
what would be a good description for it? 
"c_cpp.taskDefinitions.options.description": "Additional options including cwd and env."

Reply:
If there are additional properties supported in here, you need to list them out.  I believe you use `"properties": { }`